IGNITE-15074 : Fix documentation for support encrypted snapshots (#9238)
apache/ignite: docs/_docs/snapshots/snapshots.adoc

* Snapshotting of specific caches/tables is unsupported. You always create a full cluster snapshot.
* Caches/tables that are not persisted in Ignite Persistence are not included into the snapshot.
* Encrypted caches in the snapshot must be encrypted with same master key.
* You can have only one snapshotting operation running at a time.

> * Any snapshot operation is unallowed during master key change and/or cache group key change.

Reviewer: ```suggestion
* The snapshot operation is prohibited during a master key change and/or cache group key change.
```

Author: Fixed
